starboard/common/thread.cc

```diff
@@ -79,7 +79,11 @@ std::atomic_bool* Thread::joined_bool() {
 
 void* Thread::ThreadEntryPoint(void* context) {
   Thread* this_ptr = static_cast<Thread*>(context);
+#if defined(__APPLE__)
```

Comment:
QQ - is it better to use this `defined(__APPLE__)` (which I think
is a compiler defined thing) or `BUILDFLAG(IS_APPLE)` (which will
be a Chromium GN-originated thing?

Reply:
I'm using `__APPLE__` here because I thought `//starboard` wasn't supposed to depend on `//base` or `//build`, so `BUILDFLAG(IS_APPLE)` wasn't an option.